fic/provider.go

```diff
@@ -180,16 +180,20 @@ func Provider() terraform.ResourceProvider {
 		DataSourcesMap: map[string]*schema.Resource{},
 
 		ResourcesMap: map[string]*schema.Resource{
-			"fic_eri_firewall_component_v1":               resourceEriFirewallComponentV1(),
-			"fic_eri_nat_component_v1":                    resourceEriNATComponentV1(),
-			"fic_eri_nat_global_ip_address_set_v1":        resourceEriNATGlobalIPAddressSetV1(),
-			"fic_eri_port_to_port_connection_v1":          resourceEriPortToPortConnectionV1(),
-			"fic_eri_port_v1":                             resourceEriPortV1(),
-			"fic_eri_router_to_ecl_connection_v1":         resourceEriRouterToECLConnectionV1(),
-			"fic_eri_router_paired_to_port_connection_v1": resourceEriRouterPairedToPortConnectionV1(),
-			"fic_eri_router_single_to_port_connection_v1": resourceEriRouterSingleToPortConnectionV1(),
-			"fic_eri_router_to_uno_connection_v1":         resourceEriRouterToUNOConnectionV1(),
-			"fic_eri_router_v1":                           resourceEriRouterV1(),
+			"fic_eri_firewall_component_v1":                   resourceEriFirewallComponentV1(),
+			"fic_eri_nat_component_v1":                        resourceEriNATComponentV1(),
+			"fic_eri_nat_global_ip_address_set_v1":            resourceEriNATGlobalIPAddressSetV1(),
+			"fic_eri_port_to_port_connection_v1":              resourceEriPortToPortConnectionV1(),
+			"fic_eri_port_v1":                                 resourceEriPortV1(),
+			"fic_eri_router_to_ecl_connection_v1":             resourceEriRouterToECLConnectionV1(),
+			"fic_eri_router_paired_to_port_connection_v1":     resourceEriRouterPairedToPortConnectionV1(),
+			"fic_eri_router_single_to_port_connection_v1":     resourceEriRouterSingleToPortConnectionV1(),
+			"fic_eri_router_to_uno_connection_v1":             resourceEriRouterToUNOConnectionV1(),
+			"fic_eri_router_v1":                               resourceEriRouterV1(),
+			"fic_eri_port_to_azure_microsoft_connection_v1":   resourceEriPortToAzureMicrosoftConnectionV1(),
```

Comment:
How about change the order as alpha-numeric, like ECL provider.

Reply:
Fixed!